packages/fluent-theme/src/private/FluentThemeProvider.tsx

```diff
@@ -38,7 +39,7 @@ const FluentThemeProvider = ({ children, variant = 'fluent' }: Props) => (
     <WebChatTheme>
       <TelephoneKeypadProvider>
         <ThemeProvider activityMiddleware={activityMiddleware} sendBoxMiddleware={sendBoxMiddleware}>
-          {children}
+          <WebChatDecorator>{children}</WebChatDecorator>
```

Comment:
Maybe call it `<FluentThemeDecorator>`?

Reply:
Good call, the name left from the whitelabel unfinished work